package com.example.rickandmorty.core.internet

sealed class ApiResponseStatus<T>{
    class Success<T>(val data: T): ApiResponseStatus<T>()
    class Error<T>(val message: String): ApiResponseStatus<T>()
}


